src/widgets/profileEditor/profileEditorWidget.tsx

```diff
@@ -95,15 +100,15 @@ export interface ProfileEditorWidgetProps extends Omit<ProfileEditorProps, 'prof
      * 
      * It is not possible to update the primary identifier submitted at registration (email or phone number). When the primary identifier is the email address (SMS feature disabled), users can only enter a phone number and update without limit.
      */
-    fields?: (string | Field)[]
+    fields?: (string | FieldComponent)[];
```

Comment:
This is a user configurable option, it should not permits to the user to add a custom component.